src/status_im/data_store/realm/schemas/account/v16/core.cljs

```diff
@@ -36,15 +37,34 @@
              local-storage/schema
              handler-data/schema])
 
+(defn chat-by-id [realm chat-id]
+  (some-> realm
+          (.objects "chat")
+          (.filtered (str "chat-id = \"" chat-id "\""))
+          (aget 0)))
+
+(defn migrate-commands [realm content-type]
+  (some-> realm
+          (.objects "message")
+          (.filtered (str "content-type = \"" content-type "\""))
+          (.map (fn [object _ _]
+                  (let [group-id (aget object "group-id")
+                        {:keys [bot] :as content} (reader/read-string (aget object "content"))]
+                    (when-not bot
+                      (let [chat-id  (aget object "chat-id")
+                            chat     (chat-by-id realm chat-id)
+                            group?   (aget chat "group-chat")
+                            bot-name (if group?
```

Comment:
Isn't this dangerous assumption to make ? What if we are generating for preview for command/command-request for example for command from console and not transactor-* ?

Reply:
for the command :bot will not be empty. This is the case only for /send and /request commands atm 
It's for sure dangerous, code is ugly, etc. Though it fixes this issue...

Reply:
Maybe add some comments then? Let's try to reduce debt for future generations ;)

Reply:
yes i can add comment, though migrations are one time thing, not sure that comment will be useful